### DISCUSSION TITLE
FT-857 + FLRig + WLGate +JTDX

### DISCUSSION THREAD

**Discussion by R1BLH** [wavelog/wavelog#575] [WL Gate] [answered]
I tried to set up a bunch of FT-857 + FLRig + WLGate+JTDX. At first glance everything works. But there are a few things that prevent me from using it properly.

1. the main problem: delay in switching on the transmission when commanded from JTDX. Most likely this is a problem with the FLRig settings. 

2. There is no UDP port server number setting in WLGate and no way to forward received UDP messages. I am using a chain of JTDX messages between different software.

3. There is no way to disable QSO logging in WLGate. I use Gridtracker for logging and visualization.

 I think points 2 and 3 can be avoided by simply excluding WLGate from the JTDX message chain (not sending messages to the WLGate UDP port). But the main problem is the delay in command execution by FLRig. It was minimal when using OmniRig 1.2. Can anyone tell me what to look out for when configuring FLRig?

Mea culpa!
It looks like I found the cause of this problem.
The VFO switch command from FLRig to FT-857 works strange.
It switches the VFO three times (A->B switching actually does A -> B -> A->B switching).

To avoid this problem when configuring the software, avoid sending the VFO change command. Specifically, use the Fake VFO option in the JTDX/WSJT-X settings.

**Comment by MikleBas** [comment]
Привет!
Я тоже отказался от Flrig+Gate - обе работают плохо и не стабильно, я вообще не понимаю для чего авторы их советуют.
У меня для работы в цифре остался GridTracker + JTDX = работает великолепно, лишние программы здесь не к месту
А в CW работает данный WaveLog + CloudCat (для определения частоты и вида модуляции) + CWTYPE для морзянки. Всё, больше ничего не нужно.
Касаемо самого журнала, я влюбился в него с первого раза и сразу, настроив все как на прошлом перешел работать на него :)
сразу видно, авторы старались!
73!

**Reply by R1BLH** [reply]
Я пока настроил, посмотрим, как оно будет работать. С CloudlogCat как раз есть проблема, которая раздражает -- передача нулевой частоты, когда трансивер в режиме передачи.
I've got it set up so far, let's see how it works. With CloudlogCat, there's only one problem that's annoying -- transmitting zero frequency when the transceiver is in transmit mode.

https://github.com/wavelog/wavelog/discussions/426

**Reply by HB9HIL** [reply]
Can we stick to english here please 😄  so everyone can read and learn 😆  would be awesome

**Reply by MikleBas** [reply]
> Can we stick to english here please 😄 so everyone can read and learn 😆 would be awesome

SORRY :))
translate
Hello!
I also abandoned Flrig+Gate - both work poorly and are unstable, I don’t understand at all what the authors advise them to do.
For digital work I still have GridTracker + JTDX = it works great, unnecessary programs have no place here
And in CW this WaveLog + CloudCat works (to determine the frequency and type of modulation) + CWTYPE for Morse code. That's it, nothing more is needed.
Regarding the magazine itself, I fell in love with it the first time and right away, having set everything up like I went to work for it in the past :)
It’s immediately obvious that the authors tried!
73!

**Reply by R1BLH** [reply]
> Can we stick to english here please 😄 so everyone can read and learn 😆 would be awesome

Oops, sorry. 
On autopilot I answered in the language of the question :)
Duplicated the translation.

**Comment by int2001** [comment]
> I tried to set up a bunch of FT-857 + FLRig + WLGate+JTDX. At first glance everything works. But there are a few things that prevent me from using it properly.
> 
> 1. the main problem: delay in switching on the transmission when commanded from JTDX. Most likely this is a problem with the FLRig settings.

indeed, FLRig is not the fastest. Maybe you file an issue at FLRig or JTDX?

> 2. There is no UDP port server number setting in WLGate and no way to forward received UDP messages. I am using a chain of JTDX messages between different software.

Fair point with the UDP-Port i put it in the Backlog for one of the next Versions. Forwarding to other Software (via UDP) was (and won't ever be) any part of Wavelog-Gate. 

> 3. There is no way to disable QSO logging in WLGate. I use Gridtracker for logging and visualization.

Uhm. Simply close it if you use a different Software? I mean: you're not logging with Gridtracker to Wavelog **and** uploading the ADIF produced by JTDX as well, didn't you?

> I think points 2 and 3 can be avoided by simply excluding WLGate from the JTDX message chain (not sending messages to the WLGate UDP port). But the main problem is the delay in command execution by FLRig. It was minimal when using OmniRig 1.2. Can anyone tell me what to look out for when configuring FLRig?

Best practise here when doing FT8: I am using the Chain TRX <-> FLRig <-> WSJT-X (and of course WLGate). Without OmniRig. Problem could(!) be: OmniRig AND FLRig are trying to access your TRX in parallel. That won't work.
 

> 
> Mea culpa! It looks like I found the cause of this problem. The VFO switch command from FLRig to FT-857 works strange. It switches the VFO three times (A->B switching actually does A -> B -> A->B switching).

Yeah. another construction site. I noticed that as well with my FT857. That's the reason i sold it. It's a bug within the Yaesu-Radio.

**Reply by MikleBas** [reply]
There is a similar situation on my ICOM718
>> Yeah. another construction site. I noticed that as well with my FT857. That's the reason i sold it. It's a bug within the Yaesu-Radio.

**Reply by R1BLH** [reply, marked as answer]
> > 1. the main problem: delay in switching on the transmission when commanded from JTDX. Most likely this is a problem with the FLRig settings.
> 
>  Maybe you file an issue at FLRig or JTDX?
> 
Yes, it looks like a bug iin the FLRig (hamlib).

> > 2. There is no UDP port server number setting in WLGate and no way to forward received UDP messages. I am using a chain of JTDX messages between different software.
> 
> Fair point with the UDP-Port i put it in the Backlog for one of the next Versions. Forwarding to other Software (via UDP) was (and won't ever be) any part of Wavelog-Gate.
> 
> > 3. There is no way to disable QSO logging in WLGate. I use Gridtracker for logging and visualization.
> 
> Uhm. Simply close it if you use a different Software? I mean: you're not logging with Gridtracker to Wavelog **and** uploading the ADIF produced by JTDX as well, didn't you?

 
I find it inconvenient to start and stop the software every time I change the mode of operation.
So I'm trying to find the optimal configuration for the software. As I wrote above, some of the points are solved by changing the chain of UDP data transfer ports from JTDX and simply excluding WLGate from it. 
Let me explain: I have logging to two services (wavelog and hamlog.online), I also have mirroring of JTDX status to my mobile phone - all this is solved by a chain of software that passes messages from JTDX and back to each other.
When operating in non-digital modulation modes, the frequency/mode data of the transceiver must be transmitted to Wavelog. Therefore WLGate+FLRig should not be switched off regardless of the modulation mode.

> Best practise here when doing FT8: I am using the Chain TRX <-> FLRig <-> WSJT-X (and of course WLGate). Without OmniRig. Problem could(!) be: OmniRig AND FLRig are trying to access your TRX in parallel. That won't work.
 

I don't think anyone in their right mind would try to run two programs that use the same serial port and expect them to run at the same time :)
I wanted to look at WLGate to get rid of the OmniRig problem (which sends zero frequency to Wavelog when transmitting, causing some inconvenience). The result was that it was impossible to run split in WSJT-X/JTDX when using VFO switching on the FT-857, but it works with imitation split on the one VFO.


> > Mea culpa! It looks like I found the cause of this problem. The VFO switch command from FLRig to FT-857 works strange. It switches the VFO three times (A->B switching actually does A -> B -> A->B switching).
> 
> Yeah. another construction site. I noticed that as well with my FT857. That's the reason i sold it. It's a bug within the Yaesu-Radio.

No, this is a bug in the FLRig or Hamlib. I have tested the FT-857 with another direct control software. VFO A/B switching works perfectly,
